modules/standard/BigInteger.chpl

```diff
@@ -1906,66 +1870,48 @@ module BigInteger {
   // **=
   operator bigint.**=(ref base: bigint, const ref exp: bigint) {
     if _local {
-      mpz_powm(base.mpz, base.mpz,  exp.mpz,  base.mpz);
+      base = base ** exp;
```

Comment:
I think this may be less efficient than other bigint operators that work in place, like `+=`.  The `**` operator will create a temporary and temporary bigints are expensive, so should be avoided whenever possible.  Ideally, you would want to call to the same methods that your new `**` is calling, using base as the method receiver and one of the arguments, e.g. `base.pow(base, exp)`, I think.